File: Programmers/level1/2016년.py
def solution(a, b):
    days = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    name = ["SUN","MON","TUE","WED","THU","FRI","SAT"]
    val = 0
    for i in range(a - 1):
        val += days[i]
    val += b
    answer = name[(val+4)%7]
    return answer

File: Programmers/level1/2016년.java
class Solution {
    public String solution(int a, int b) {
        int[] day = {31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
        String[] d = {"SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"};

        int diff = 0;
        for (int i=0;i < a - 1;i++){
            diff += day[i];
        }
        diff += b + 4;
        String answer = d[diff % 7];
        return answer;
    }
}